core/node/vm_runner/src/output_handler.rs

```diff
@@ -248,6 +251,9 @@ impl<Io: VmRunnerIo> ConcurrentOutputHandlerFactoryTask<Io> {
                     self.io
                         .mark_l1_batch_as_completed(&mut conn, latest_processed_batch)
                         .await?;
+                    METRICS
+                        .last_processed_batch
+                        .set(latest_processed_batch.0 as u64);
```

Comment:
Nit: For lossless conversions (like `u32` -> `u64` here) using `.into()` is preferable because it signals that the conversion is lossless.